app/src/main/java/com/orange/ods/app/ui/components/appbars/top/ComponentTopAppBar.kt

```diff
@@ -65,204 +42,37 @@ fun ComponentTopAppBar(variant: Variant) {
-                                bottom = dimensionResource(id = com.orange.ods.R.dimen.spacing_xs)
-                            ),
-                            text = stringResource(id = R.string.component_app_bars_top_large_code_collapsing_step_2)
-                        )
-                        CodeBackgroundColumn {
-                            TechnicalText(text = "val nestedScrollConnection = remember { topBarScrollBehavior.nestedScrollConnection }")
-                            TechnicalText(text = "Scaffold(modifier = Modifier.nestedScroll(nestedScrollConnection), ...) { ... }")
-                        }
+            ComponentCustomizationBottomSheetScaffold(
+                bottomSheetScaffoldState = rememberBottomSheetScaffoldState(),
+                bottomSheetContent = {
+                    when (variant) {
+                        Variant.AppBarsTopLarge -> LargeTopAppBarBottomSheetContent()
+                        Variant.AppBarsTopSearch -> TopAppBarNavigationIconSetup()
+                        else -> RegularTopAppBarBottomSheetContent()
                     }
                 }
-
-                if (isLargeCollapsible) {
-                    BlinkingChevronDown(modifier = Modifier.padding(vertical = dimensionResource(id = com.orange.ods.R.dimen.spacing_s)))
-                    OdsTextBody2(text = stringResource(id = R.string.component_app_bars_top_large_scrolling_downward))
+            ) {
+                when (variant) {
+                    Variant.AppBarsTopLarge -> LargeTopAppBarContent()
+                    Variant.AppBarsTopSearch -> SearchTopAppBarContent()
+                    else -> RegularTopAppBarContent()
                 }
             }
         }
     }
 }
 
 @Composable
-private fun CustomizationBottomSheetContent(customizationState: TopAppBarCustomizationState, isLarge: Boolean) {
-    with(customizationState) {
-        if (isLarge) {
-            Subtitle(textRes = R.string.component_app_bars_top_large_scroll_behavior, horizontalPadding = true)
-            OdsChoiceChipsFlowRow(
-                value = scrollBehavior.value,
-                onValueChange = { value -> scrollBehavior.value = value },
-                modifier = Modifier.padding(horizontal = dimensionResource(id = com.orange.ods.R.dimen.spacing_m)),
-                chips = listOf(
-                    OdsChoiceChip(
-                        text = stringResource(id = R.string.component_app_bars_top_large_scroll_behavior_none),
-                        value = TopAppBarCustomizationState.ScrollBehavior.None
-                    ),
-                    OdsChoiceChip(
-                        text = stringResource(R.string.component_app_bars_top_large_scroll_behavior_collapsible),
-                        value = TopAppBarCustomizationState.ScrollBehavior.Collapsible
-                    )
-                )
-            )
-        }
+fun TopAppBarNavigationIconSetup() {
```

Comment:
I did not find any other method with `Setup`, maybe we could rename this method to `TopAppBarNavigationIconCustomization`?